examples/nodejs/baseai/memory/ai-agent-memory/index.ts

```diff
@@ -10,6 +10,14 @@ const memoryAiAgentMemory = (): MemoryI => ({
 		deployedAt: '',
 		embeddedAt: '',
 	},
+	document: {
+		meta: doc => {
+			return {
+				name: doc.name,
+				path: doc.path,
```

Comment:
A url example will help with a base url of langbase.com/docs to make it clear.

Reply:
Updated it.